proposed/extended-coding-style-guide.md

```diff
@@ -546,6 +546,15 @@ class ReturnTypeVariations
 }
 ~~~
 
```

Comment:
```suggestion

When using the reference operator `&` before an argument, there MUST be a space before the operator, and there MUST NOT be a space after it, like in the previous examples.

```

Reply:
The space should imho be there only when type is present, i.e. `function foo(&$args)`  and not `function foo( &$args)`

Reply:
I guess, "there MUST be a space before the operator" should be omitted since that space is covered by "space after comma" rule and, as @JanTvrdik said, is wrong in case argument is first.

Reply:
Yup, right.